frontend/components/slot/viz/SlotVizTooltip.vue

```diff
@@ -164,7 +164,7 @@ const data = computed(() => {
       <div class="with-duties">
         <div class="rows">
           <div class="row network">
-            <BcFormatNumber :text="data.networkLabel" />
+            <BcMiniParser :input="data.networkLabel" :numbers="{ slotNum: props.data.slot }" :links="{ slotUrl: `/slot/${props.data.slot}` }" />
```

Comment:
:thinking: I do not find this component

Reply:
Yes it is in the "research BIDS" created this morning.
But you can see that this version A is simpler than the version B (version B works, you can review/test it).